### ISSUE TITLE
Support for replying to a conversation

### ISSUE THREAD

**Issue by ebaizel** [tarunc/intercom.io#30]
Great library!  Any chance it can include [replying to a conversation](https://developers.intercom.io/docs/replying-to-a-conversation)?  

Would be happy to contribute too.

**Comment by tarunc** [comment]
Its included in [`v1.2.0`](https://www.npmjs.com/package/intercom.io) (the most recent version on NPM). Just update the version using `npm update intercom.io` The conversation methods are:
```
* intercom.listConversations
* intercom.getConversation
* intercom.replyConversation
* intercom.markConversationAsRead
* intercom.closeConversation
```

You should be able to reply to a conversation using:
```javascript
intercom.replyConversation({
  "id": "147", // Note that without the id parameter, 
               // it will automatically reply to the last conversation (just like intercom)
  "intercom_user_id": "536e564f316c83104c000020",
  "body": "Let me know if you have any more questions!",
  "type": "admin",
  "admin": "1234",
  "message_type": "comment"
}).then(function(res) {
  // res is **JSON** response
  // In this case:
  // {
  //    "type": "conversation",
  //    "id": "147",
  //    ... Rest of the Conversation Object ...
  //    See https://developers.intercom.io/docs/conversation-model
  // }
}, function(err) {
  // err is an error object if there was an error
});
```

For complete documentation with how to deal with conversations, see [Intercom API Docs](https://developers.intercom.io/docs/conversations).

Let me know if you need help using any of the methods.

**Comment by ebaizel** [comment]
:+1:  looks like exactly what i was looking for.
